### DISCUSSION TITLE
Can this be extended to orbital DFT?

### DISCUSSION THREAD

**Discussion by nkarve** [profess-dev/profess-ad#2] [Q&A]
First of all this is an excellent project, I came across it while trying to develop my own autograd-based DFT solver for periodic systems (great minds think alike!). 

I noticed your version is currently only for orbital-free DFT - how hard would it be to extend this to orbital-based DFT? The specific toy example to consider for concreteness is simply the Schrödinger equation for a particle in potential _V_, where the object to be optimised via gradient descent is no longer the density directly but rather the coefficients in reciprocal space.

**Comment by wcwitt** [comment]
It's definitely something we're thinking about, but not on the immediate horizon. You might be interested in [DFTK](https://github.com/JuliaMolSim/DFTK.jl) or differentiable [PySCF](https://arxiv.org/abs/2207.13836).

**Comment by nkarve** [comment]
Thanks for the links!
